feature-libs/quote/components/header/overview/quote-header-overview.component.ts

```diff
@@ -41,16 +46,25 @@ export class QuoteHeaderOverviewComponent {
   quoteDetails$: Observable<Quote> = this.quoteFacade.getQuoteDetails();
   iconTypes = ICON_TYPE;
   editMode = false;
+  enablePurchaseOrderNumber =
+    this.featureToggles.enableQuotePurchaseOrderNumber;
 
   protected defineQuoteMetaData(event: SaveEvent): QuoteMetadata {
     let metaData: QuoteMetadata = {};
     if (Object.getOwnPropertyNames(event).length >= 1) {
-      const [name, description] = [event.name, event.description];
+      const [name, description, purchaseOrderNumber] = [
+        event.name,
+        event.description,
+        event.purchaseOrderNumber,
+      ];
```

Comment:
Is this expression trying to do a de-structure of `event`?
```suggestion
      const {name, description, purchaseOrderNumber} = event;
```

Reply:
changed